python/ppml/src/bigdl/ppml/fl/pytorch/pipeline.py

```diff
@@ -0,0 +1,90 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import logging
+from numpy import ndarray
+import torch
+from torch import nn
+import torch
+from bigdl.ppml.fl.pytorch.fl_client import FLClient
+from torch.utils.data import DataLoader
+from bigdl.ppml.fl.pytorch.protobuf_utils import tensor_map_to_ndarray_map
+
+class PytorchPipeline:
```

Comment:
instead of pipeline, maybe implement an Orca-like Estimator (e.g., `ppml.fl.Estimator.from_torch`)?

Reply:
Would be refined in next PRs